lib/vsc/mympirun/rm/sched.py

```diff
@@ -256,14 +251,11 @@ def set_mpinodes(self):
         """
 
         res = []
-        if getattr(self.options, 'double', False):
-            res = self.nodes * 2
+        if self.options.hybrid is not None:
```

Comment:
@Caylo positive logic is easier to follow, so use `if self.options.hybrid is None` and flip `if`/`else` around